translations/en/translation.json

```diff
@@ -22,17 +22,24 @@
     },
     "globalErrors": {
       "reference": {
+        "cannotAccess": "\nError ▶️ \"{{symbol}}\" is used before initialization. Make sure you have initialized the variable before using it.\n\n{{location}}\n\nFor more: {{url}}",
         "notDefined": "There's an error due to \"{{symbol}}\" not being defined in the current scope {{location}}.\n\nIf you have defined it in your code, you should check its scope, spelling, and letter-casing (JavaScript is case-sensitive). For more:\n{{url1}}\n{{url2}}"
       },
       "stackSubseq": "▶️ Called from line {{line}} in \"{{func}}\" in {{file}} ({{location}})\n\n",
       "stackTop": "▶️ Error at line {{line}} in \"{{func}}\" in {{file}} ({{location}})\n\n",
       "syntax": {
-        "invalidToken": "There's a syntax error due to a symbol that JavaScript doesn't recognize or didn't expect at its place.\nFor more: {{url}}",
-        "unexpectedToken": "There's a syntax error due to a symbol that wasn't expected at its place.\nUsually this is due to a typo. Check the line number in the error below for anything missing/extra.\nFor more: {{url}}"
+        "badReturnOrYield": "\nSyntax Error ▶️ `return` / `yield` lies outside of a function. Make sure you’re not missing any brackets, so that `return` / `yield` lies inside a function.\nFor more: {{url}}",
```

Comment:
Here I would suggest `"\nSyntax Error ▶️ `return` / `yield` is outside of a function. Make sure you’re not missing any brackets, so that `return` / `yield` is inside a function.\nFor more: {{url}}"`

Reply:
I have made the changes please have a look